apps/prairielearn/src/pages/studentAssessment/studentAssessment.ts

```diff
@@ -73,16 +73,16 @@ router.get(
     if (!res.locals.assessment.group_work && res.locals.assessment.type === 'Homework') {
       const time_limit_min = null;
       const client_fingerprint_id = await getClientFingerprintId(req, res);
-      const assessment_instance_id = await makeAssessmentInstance(
-        res.locals.assessment.id,
-        res.locals.user.user_id,
-        res.locals.assessment.group_work,
-        res.locals.authn_user.user_id,
-        res.locals.authz_data.mode,
+      const assessment_instance_id = await makeAssessmentInstance({
+        assessment_id: res.locals.assessment.id,
```

Comment:
Nit: consider passing in the entire `assessment` object, which would let you get rid of the `group_work` property below.

Reply:
I considered this, but it would require the admin query to be a bit more complex. I'd prefer not to do it here.

Reply:
That said, there might be another cleaner way to do it... Let me play around.

Reply:
I'm happy to save that for a potential future PR if you'd prefer.

Reply:
Done in 57e83d3ad278e650da34248488c0389223a9bdbe.

Reply:
Looks great!